MyLibs/writeOutFish.h

```diff
@@ -25,6 +25,13 @@
 * Created on: Jan. 1, 2018
 */
 
+// TODO: redefine this class, think it clear the relationship between
+// this WriteOut class and the ExperimentData class
+// TODO: unify the input from one place (file, script or command window)
```

Comment:
This was done by you. Good job. Now all the inputs are required through the command window. 
@WanzhuoShi 👍